pep-A.rst

```diff
@@ -71,7 +67,7 @@ Rationale
 
 The mechanism of handling missing standard library modules through the use of the ``.missing.py`` files was chosen due to its advantages both for CPython itself and for Linux and other distributions that are packaging it.
 
-The missing pieces of standard library modules can be subsequently installed simply by putting the module files in their appropriate location, which will then take precedence over the corresponding ``.missing.py`` files.  This makes installation simple for Linux package managers.
+Missing pieces of the standard library modules can be subsequently installed simply by putting the module files in their appropriate location. They will then take precedence over the corresponding ``.missing.py`` files.  This makes installation simple for Linux package managers.
```

Comment:
I believe the first "The" should be there, it sounds more natural as we've already referred to the missing modules in this section.